### PR TITLE
Fixed doubling of assets JSON.

### LINKED ISSUE #34: JSON within JSON in Export Data
When the `JSONString` builds in `export.js`, it adds the `assets` object twice (same data) nested within itself. Not sure why this is happening, but we've got a suspicion it has to do with async.

### PR DESCRIPTION
**Pull request by purukaushik**
Fixes #34

### CODE REVIEW

File: js/export.js [hunk 1]
```diff
@@ -20,15 +20,12 @@ document.addEventListener('DOMContentLoaded', function() {
...
+
 
 				// clear anything that may have been in our textarea
-				var assetBenchmarks = result.assetBenchmarks;
-				for (var record in assetBenchmarks) {
-			        JSONString += JSON.stringify(assetBenchmarks[record]) + ",";
-		        }
-		        JSONString += JSONString.substring(0, JSONString.length-1) + "]}]}";
-		        dataOutput.textContent = JSONString;
+				var dict = { "assets": result.assetBenchmarks}
```
Comment (FrancescoSTL) [inline js/export.js:26]: nit: indentation

Comment (purukaushik) [reply js/export.js:26]: snapple. will fix

Comment (FrancescoSTL) [reply js/export.js:26]: swag. Reviewing functionality rq

**Comment by FrancescoSTL** [comment]
Working for me. r+ w/c

**Comment by FrancescoSTL** [comment]
@purukaushik go ahead and add the hash fix to this PR

**Comment by FrancescoSTL** [comment]
wait... thats the other repo. Nevermind. 🤔

**Comment by purukaushik** [comment]
yay!

**Pull request merged**